src/Infrastructure.VS/Resources.resx

```diff
@@ -120,6 +120,9 @@
   <data name="FailedToCalculateVsVersion" xml:space="preserve">
     <value>Failed to calculate VS version, error: {0}</value>
   </data>
+  <data name="NoGitFolder" xml:space="preserve">
+    <value>Workspace is not under git control</value>
```

Comment:
`Solution` is a more normal term in VS than `Workspace` (does VS use the word `Workspace` anywhere? I can't find it on any of the menus).

How about _Solution/folder is not in a git repository_ ?